tests/components/tibber/test_common.py

```diff
@@ -0,0 +1,36 @@
+"""Test common."""
+import datetime as dt
+from unittest.mock import AsyncMock
+
+_CONSUMPTION_DATA_1 = [
```

Comment:
If we use it in another module it should not be prefixed with underscore.
```suggestion
CONSUMPTION_DATA_1 = [
```